src/slider/slider.css

```diff
@@ -0,0 +1,176 @@
+@value dark from "../global/variables_dark.css";
+
+:root {
+  --ring-slider-thumb-color: var(--ring-content-background-color);
```

Comment:
How do you think, doesn't it make sense to define these variables not on the `:root` but on Slider container `.slider` instead? That way, it would be more local and easier to debug

Reply:
Good idea, this does make sense, fixed!